modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheDatabaseSharedManager.java

```diff
@@ -1462,6 +1462,9 @@ private void prepareIndexRebuildFuture(int cacheId) {
         GridQueryProcessor qryProc = cctx.kernalContext().query();
 
         if (qryProc.moduleEnabled()) {
+            CountDownCallback rebuildIndexesCompleteCntr =
```

Comment:
Why thrid argument in `CoundDownCallback` should be 1? Please add comment to code.